### PR TITLE
Upgrade uv-metrics tutorial

### PR DESCRIPTION
**Pull request by sritchie**
This PR upgrades the tutorial to be compatible with the latest changes in https://pypi.org/project/uv-metrics/.

### CODE REVIEW

**Comment by sritchie** [comment]
@ajslone , this should be a good test of the MLFlow integration you're working on!

**Comment by sritchie** [comment]
@ajslone please feel free to merge if you think this looks good.

File: tutorials/uv-metrics/requirements.txt [hunk 1]
```diff
@@ -1,9 +1,2 @@
+# Get it done
```
Comment (ajslone) [inline tutorials/uv-metrics/requirements.txt:1]: So uv/master is not compatible with this? I thought those changes were in now?

File: tutorials/uv-metrics/trainer/train.py [hunk 2]
```diff
@@ -420,8 +420,8 @@ def model_main(activation='relu', width=1000, depth=2, lr=0.5, **kwargs):
   model = build_model(activation, width, depth)
   optimizer = tf.optimizers.SGD(lr)
 
-  with r.start_run():
-    r.log_params({
+  with uv.start_run():
```
Comment (ajslone) [inline tutorials/uv-metrics/trainer/train.py:423]: This is no longer a context manager, right?

File: tutorials/uv-metrics/requirements.txt (hunk 1 above)
Comment (sritchie) [reply tutorials/uv-metrics/requirements.txt:1]: whoops, sorry to leave this in. yes, it is now compatible

**Comment by sritchie** [comment]
@ajslone this should work with UV master, so maybe for now confirm that against your work - would you mind giving it an update and merge if it seems like it will work?

**Comment by ajslone** [comment]
There are a couple of issues right now, the primary one is that the parameters do not get reported using `uv.report_params` as the active reporter has not been set to an `MLFlowReporter`, so they are being set on a `NullReporter`. I can push a change for this.

**Pull request merged**